src/Arguments/AppService/BaseAppServiceArguments.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using AzureMcp.Models.Argument;
+using System.Text.Json.Serialization;
+
+namespace AzureMcp.Arguments.AppService;
+
+public class BaseAppServiceArguments : SubscriptionArguments
+{
+
+}
```

Comment:
I believe this can be simplified to using only a `;` to represent the class body like the other argument classes.